src/sentry/notifications/helpers.py

```diff
@@ -0,0 +1,150 @@
+from collections import defaultdict
```

Comment:
This file looks like it's basically entirely typed - could you add it to `mypy.ini` (assuming it's not throwing a bunch of type errors for other reasons)